zproject/settings.py

```diff
@@ -809,6 +809,8 @@ def get_secret(key):
             'third/jquery-autosize/jquery.autosize.js',
             'third/jquery-perfect-scrollbar/js/perfect-scrollbar.js',
             'third/lazyload/lazyload.js',
+            'node_modules/moment/moment.js',
+            'third/moment-timezone/moment-timezone-with-data.min.js',
```

Comment:
Does `third/moment-timezone/moment-timezone-with-data.min.js` not exist in NPM?  That'd be the preferred way to do it.  If that isn't possible, let's check in the non-minified version of this library -- our tools will minify it, and importantly the Git history will be much more useful than if we check in a minified third-party thing.